pithos/plugins/mediakeys.py

```diff
@@ -92,13 +97,15 @@ def prepare_keybinder():
                     self.prepare_complete()
 
         def on_new_finish(source, result, data):
+            nonlocal de
```

Comment:
I'm lost, where is `de` actually set?

Reply:
https://github.com/JasonLG1979/pithos/blob/876e08a60d8541b96318fa4d6e33f966e09a8db5/pithos/plugins/mediakeys.py#L134 or https://github.com/JasonLG1979/pithos/blob/876e08a60d8541b96318fa4d6e33f966e09a8db5/pithos/plugins/mediakeys.py#L112